src/DIRAC/__init__.py

```diff
@@ -198,3 +213,14 @@ def abort(exitCode, *args, **kwargs):
   except OSError:
     gLogger.exception('Error while executing DIRAC.abort')
     os._exit(exitCode)
+
+
+def extension_metadata():
+  return {
+      "priority": 0,
+      "setups": {
+          "DIRAC-Certification": {
```

Comment:
This isn't used yet but it will be in my follow up PR to this one. I didn't mean to include it here but I see no harm in leaving it.